app/api/annotations.py

```diff
@@ -64,6 +67,9 @@ def get(self, annotation_id):
         if annotation is None:
             return {"message": "Invalid annotation id"}, 400
 
+        image = current_user.images.filter(id=annotation.image_id, deleted=False).first()
+        image.update(set__is_modified=False)
```

Comment:
Why do we need to update the status of an image if we are returning an annotation?

Reply:
My bad, I'll make the changes tomorrow.

Reply:
When the image has no annotations, the endpoint is [this one](https://github.com/AdanMora/coco-annotator/blob/185ca53967928f10e2775f4d933110837e84de1e/app/api/images.py#L110), if I put the same code used [here](https://github.com/AdanMora/coco-annotator/blob/185ca53967928f10e2775f4d933110837e84de1e/app/api/images.py#L193) when there is annotations, if I open the image, it will load the thumbnail instead of the original image.

Reply:
I'm slightly confused. Why are we putting the same code at `/<image_id>/` where `/<image_id>/thumbnail` is?

Reply:
You know your code better than me so tell me if I'm wrong, when the image doesn't have annotations `/<image_id>/` is called, but when has annotations `/<image_id>/thumbnail` is called in order to get the preview of the annotations, that's why you should put the same code, but if you do it, when you try to open the image and use the tools for annotations, the thumbnail is opened instead of the original image.

Reply:
You cant combine them, since one returns an image with annotations and the other just returns the image

Reply:
Exactly, so, there is a way to store the thumbnail without annotations and load the original image at the same time?

Reply:
Yes, but why would you want to do that?